Library/Formula/emacs.rb

```diff
@@ -39,6 +40,7 @@ class Emacs < Formula
   depends_on "imagemagick" => :optional
   depends_on "mailutils" => :optional
   depends_on "glib" => :optional
+  depends_on "libxml2" => :optional
```

Comment:
No worries - I was doing this PR for linuxbrew before and they told me if I add this change to homebrew it gets automatically merged every week so the PR in linuxbrew has been closed again without being merged.

Following your comment I'm going to remove this line as it's not necessary for OS X...